resources/views/livewire/all-projects.blade.php

```diff
@@ -0,0 +1,221 @@
+<x-slot name="title">
+    {{ __('Projects') }}
+</x-slot>
+
+<x-slot name="header">
+    <h1 id="browse-all-projects">
+        {{ __('Browse all projects') }}
+    </h1>
+</x-slot>
+
+<div>
+    <form class="stack" wire:submit.prevent="search">
+        <x-hearth-label for="searchQuery" :value="__('Search')" />
+        <div class="repel my-0">
+            <x-hearth-input name="searchQuery" type="search" wire:model.defer="searchQuery" wire:search="search" />
+            <button>{{ __('Search') }}</button>
+        </div>
+    </form>
+
+    <div class="search" role="alert">
+        @if ($searchQuery)
+            <p class="h4">
+                {{ __(':count results for ":searchQuery".', ['count' => $projects->total(), 'searchQuery' => $searchQuery]) }}
+            </p>
+        @elseif ($statuses ||
+            $seekings ||
+            $seekingGroups ||
+            $initiators ||
+            $meetingTypes ||
+            $locations ||
+            $compensations ||
+            $sectors ||
+            $impacts ||
+            $recruitmentMethods)
+            <p class="h4">
+                {{ __(':count projects match your applied filters', ['count' => $projects->total()]) }}
+            </p>
+        @endif
+    </div>
+
+    <div class="stack with-sidebar with-sidebar:2/3">
+        <div class="filters">
+            <h2 class="visually-hidden">{{ __('Filters') }}</h2>
+            <x-expander :level="3">
+                <fieldset class="filter__options field @error('status') field--error @enderror">
+                    <x-slot name="summary">{{ __('Status') }}</x-slot>
+                    <ul class="mb-4" role="list">
```

Comment:
Let's make this a style specific to filter `<ul>` elements in our stylesheet (maybe make and import a new stylesheet partial in `components/_filters.css` and import it into `app.css`):

```css
.filters .expander .ul {
  margin-block-end: var(--space-4);
}
```